.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Question
+about: Ask a question about depstat
+title: ''
+labels: kind/question
```

Comment:
Again, please don't use GitHub issues for questions or discussions. Support questions should go to discuss.kubernetes.io.

You can set up a config.yml for this - https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/config.yml